packages/sn-client-core/src/Schemas/SchemaStore.ts

```diff
@@ -57,7 +57,9 @@ export class SchemaStore {
     }
     let schema = this.schemas.find(s => s.ContentTypeName === contentTypeName) as Schema
     if (!schema) {
-      return this.getSchema(GenericContent)
+      const genericContent = this.getSchema(GenericContent)
+      genericContent.DisplayName = contentTypeName
```

Comment:
While I see why you added this change I'm not sure if this is the right way. Lets discuss this on Monday